test/Ocean/HydrostaticBoussinesq/test_ocean_gyre_long.jl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env julia --project
+using Test
+
+include("../../../experiments/OceanBoxGCM/simple_box.jl")
+ClimateMachine.init()
+
+#################
+# RUN THE TESTS #
+#################
+const FT = Float64
```

Comment:
will these toplevel constants be an issue if all the tests are evaluated in the same scope?  it might be better to include them in the testset?

Reply:
ah that sounds like a good idea. it hasn't caused an issue previously and we have a few existing tests that use this setup

Reply:
I think I was trying to cheat with some of these and actually use them as globals

Reply:
This shows up other places: https://github.com/CliMA/ClimateMachine.jl/issues/1368

Reply:
okay, I managed to work around everything but the `const FT`. Would be nice if we could set `FT` as part of something like `ClimateMachine.init()` so we don't have to pass it through every single function.